spot-ml/src/main/scala/org/apache/spot/SpotSparkLDAWrapper.scala

```diff
@@ -83,7 +100,7 @@ object SpotSparkLDAWrapper {
     //Convert to DistributedLDAModel to expose info about topic distribution
     val distLDAModel = ldaModel.asInstanceOf[DistributedLDAModel]

```

Comment:
Would be nice, I can try. That means you really want to get ride of online optimizer and just keep "em".